tests/e2e/specs/edit-posts.test.js

```diff
@@ -40,19 +43,20 @@ describe( 'Edit Posts', () => {
 		expect( postTitle.length ).toBe( 1 );
```

Comment:
We can also use locator assertions here, e.g. `expect(firstPost).toHaveText(title)`